src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -1452,6 +1465,49 @@ private static String getValueForId(int id, String fieldName, String namespace,
         return value;
     }
 
+    /**
+     * Parse a list field in the schedule exceptions table (dates, added_service, or removed_service) into a list of string.
```

Comment:
nit. "... array of strings."

Reply:
changed in https://github.com/conveyal/gtfs-lib/pull/376/commits/f035d18dc1721f03bfea4a1d5435184ab22f01e5